test/functional/apps/discover/_field_data.js

```diff
@@ -36,7 +36,8 @@ export default function ({ getService, getPageObjects }) {
       // delete .kibana index and update configDoc
       await kibanaServer.uiSettings.replace({
         'dateFormat:tz': 'UTC',
-        'defaultIndex': 'logstash-*'
+        'defaultIndex': 'logstash-*',
+        'accessibility:disableAnimations': 'true',
```

Comment:
This would not be necessary if this was moved from `replace` to `update`, as `accessibility:disableAnimations` is true by default thanks to https://github.com/elastic/kibana/blob/master/test/functional/config.js#L129-L133